app/javascript/modCenter/singleArticle/__tests__/singleArticle.test.jsx

```diff
@@ -39,37 +39,24 @@ describe('<SingleArticle />', () => {
   });
 
   it('renders the article title', () => {
-    const { queryByText } = render(
+    const articleProps = getTestArticle();
```

Comment:
(not blocking) - Looking at the specs in this file, I don't think there's any need for `getTestArticle` to be a function 🤔  If it wasn't, then we wouldn't need to call it every spec like this... but then it's not really related to this PR so could maybe be a later enhancement 🙂

Reply:
Totally agree!